docs/source/llm/export-custom-llm.md

```diff
@@ -0,0 +1,424 @@
+# Exporting custom LLMs
+
+If you have your own PyTorch model that is an LLM, this guide will show you how to manually export and lower to ExecuTorch, with many of the same optimizations as covered in the previous `export_llm` guide.
+
+This example uses Karpathy’s [nanoGPT](https://github.com/karpathy/nanoGPT), which is a minimal implementation of
+GPT-2 124M. This guide is applicable to other language models, as ExecuTorch is model-invariant.
+
+
+## Exporting to ExecuTorch (basic)
+
+Exporting takes a PyTorch model and converts it into a format that can run efficiently on consumer devices.
+
+For this example, you will need the nanoGPT model and the corresponding tokenizer vocabulary.
+
+::::{tab-set}
+:::{tab-item} curl
+```
+curl https://raw.githubusercontent.com/karpathy/nanoGPT/master/model.py -O
+curl https://huggingface.co/openai-community/gpt2/resolve/main/vocab.json -O
+```
+:::
+:::{tab-item} wget
+```
+wget https://raw.githubusercontent.com/karpathy/nanoGPT/master/model.py
+wget https://huggingface.co/openai-community/gpt2/resolve/main/vocab.json
+```
+:::
+::::
+
+To convert the model into a format optimized for standalone execution, there are two steps. First, use the PyTorch
+`export` function to convert the PyTorch model into an intermediate, platform-independent intermediate representation. Then
+use the ExecuTorch `to_edge` and `to_executorch` methods to prepare the model for on-device execution. This creates a .pte
+file which can be loaded by a desktop or mobile application at runtime.
+
+Create a file called export_nanogpt.py with the following contents:
+
+```python
```

Comment:
so much code (I know you are just moving here) in an `.md` is just waiting to get outdated. Is there an alternative like a notebook or something  (which is also in CI)?

Reply:
Yeah, it'd be good to put into a notebook maybe